tests/blas_test.cpp

```diff
@@ -317,29 +318,29 @@ void initFields(prec_pair_t prec_pair)
 void freeFields()
 {
   // release memory
-  delete vD;
-  delete wD;
-  delete xD;
-  delete yD;
-  delete zD;
-  delete voD;
-  delete woD;
-  delete xoD;
-  delete yoD;
-  delete zoD;
-  delete xmD;
-  delete ymD;
-  delete zmD;
-  delete xmoD;
-  delete ymoD;
-  delete zmoD;
+  vD.reset();
```

Comment:
Is there a particular reason we're explicitly calling `reset()` as opposed to letting `unique_ptr` "do its thing"?

Reply:
Yes. `unique_ptr` does not go out of scope before `endQuda` and that will complain about memory leaks. 
So for now we still need to reset this to avoid the warning.